plugins/hiscore/init.lua

```diff
@@ -46,8 +46,10 @@ function hiscore.startplugin()
 		emu.print_verbose( "hiscore: config found" );
 		local _conf = {}
 		for line in io.lines(config_path) do
-		  token, value = string.match(line, '([^ ]+) ([^ ]+)');
-		  _conf[token] = value;
+		  token, spaces, value = string.match(line, '([^ ]+)([ ]+)([^ ]+)');
```

Comment:
Wouldn't this be simpler as:
```
token, value = string.match(line, '([^ ]+) +([^ ]+)');
```
There's no reason to use a character class for the spaces, and no reason to use a capture group for them.

Reply:
i agree, committed to my other pending PR: https://github.com/mamedev/mame/pull/6130